test/integration/targets/docker_container/tasks/tests/options.yml

```diff
@@ -592,6 +592,62 @@
     - devices_3 is not changed
     - devices_4 is changed
 
+####################################################################
+## device_read_bps #################################################
+####################################################################
+
+- name: device_read_bps
+  docker_container:
+    image: alpine:3.8
+    command: '/bin/sh -c "sleep 10m"'
+    name: "{{ cname }}"
+    state: started
+    device_read_bps:
+      - path: /dev/random
+        rate: 20mb
+      - path: /dev/urandom
+        rate: 10kb
+  register: device_read_bps_1
+
+- name: device_read_bps (idempotency)
+  docker_container:
+    image: alpine:3.8
+    command: '/bin/sh -c "sleep 10m"'
+    name: "{{ cname }}"
+    state: started
+    device_read_bps:
+      - path: /dev/random
+        rate: 20mb
+      - path: /dev/urandom
+        rate: 10kb
+  register: device_read_bps_2
+
+- name: device_read_bps (changed)
+  docker_container:
+    image: alpine:3.8
+    command: '/bin/sh -c "sleep 10m"'
+    name: "{{ cname }}"
+    state: started
+    device_read_bps:
+      - path: /dev/random
+        rate: 10mb
+      - path: /dev/urandom
+        rate: 5kb
+    stop_timeout: 1
+  register: device_read_bps_3
```

Comment:
You could add two more tests: one with one less entry (shouldn't change), and one with one more entry (should change). See the tests for the `devices` option for example.